starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+.pieImage{
+  background-image: url('../images/apple-pie.jpg');
```

Comment:
background image was a little small. Would be good to adjust the width of the div a little. About 60em would be good.